R/package.r

```diff
@@ -89,7 +89,9 @@ package_rd <- function(path) {
 extract_tag <- function(x, tag) {
   x %>%
     purrr::keep(inherits, tag) %>%
-    purrr::map_chr(c(1, 1))
+    unlist %>%
+    paste(collapse = " ") %>%
+    stringr::str_trim()
```

Comment:
Also if you're going to take this approach you're going to need some unit tests to verify that it actually does what you expect.